labelspark/__init__.py

```diff
@@ -39,6 +39,39 @@ def get_annotations(client, project_id, spark, sc):
     return bronze_table
 
 
+import requests  #we need to handle a large frame-by-frame dataset for videos, so we use requests
+
+
+def get_videoframe_annotations(bronze_video_labels, API_KEY, spark, sc):
+    # This method takes in the bronze table from get_annotations and produces
+    # an array of bronze dataframes containing frame labels for each project
+    bronze_video_labels = bronze_video_labels.withColumnRenamed(
+        "DataRow ID", "DataRowID")
+    koalas_bronze = bronze_video_labels.to_koalas()
+
+    # We manually build a string of frame responses to leverage our existing jsonToDataFrame code, which takes in JSON
+    headers = {'Authorization': f"Bearer {API_KEY}"}
+    master_array_of_json_arrays = []
+    master_array_of_toplevel_rows = []
+    for index, row in koalas_bronze.iterrows():
```

Comment:
I would advise against directly building the json string. 

```python
data = []
for line in response.iter_lines():
   data.append({"DataRow ID": "..."})

return json.dumps(data)
```
is a lot cleaner

Reply:
I've adjusted this to 

        data = []
        for line in response.iter_lines(): 
            data.append({"DataRow ID": row.DataRowID,
                         "Label": line.decode('utf-8')})
        massive_string_of_responses = "[" + ",".join(
            json.dumps(data)) + "]"
            
Which I believe will solve the need to build the JSON but in a cleaner manner.

Reply:
you could also do 

```python
 data = []
    for line in response.iter_lines(): 
        data.append({"DataRow ID": row.DataRowID,
                     "Label": line.decode('utf-8')})
    massive_string_of_responses = json.dumps(data)
```
right?